spec/features/publish/providers_index_spec.rb

```diff
@@ -0,0 +1,64 @@
+require "rails_helper"
+
+feature "Providers index" do
+  context "user with multiple accredited bodies" do
+    let(:current_recruitment_cycle) { find_or_create(:recruitment_cycle) }
```

Comment:
We've been writing the feature specs without using lets and encapsulating the data into their own steps so the spec is more readable. Could we do that here and for the other spec? Whichever way we decide (we can review later) we should be consistent in the approach